modules/combined/examples/periodic_strain/global_strain_pfm.i

```diff
@@ -0,0 +1,424 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 50
+  ny = 50
+  xmin = -0.5
+  xmax = 0.5
+  ymin = -0.5
+  ymax = 0.5
+[]
+
+[MeshModifiers]
+  [./cnode]
+    type = AddExtraNodeset
+    coord = '0.0 0.0'
+    new_boundary = 100
+  [../]
+[]
+
+[Variables]
+  [./u_x]
+  [../]
+  [./u_y]
+  [../]
+  [./global_strain]
+    order = THIRD
+    family = SCALAR
+  [../]
+  [./c]
+    [./InitialCondition]
+      type = FunctionIC
+      function = 'sin(2*x*pi)*sin(2*y*pi)*0.05+0.6'
+    [../]
+  [../]
+  [./w]
+  [../]
+[]
+
+[AuxVariables]
+  [./dispg_x]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./dispg_y]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./local_energy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./disp_x]
+    order = CONSTANT
```

Comment:
Those should be nodal!

Reply:
Not sure that this would be compatible with nodal variables since the global displacement is being calculated from `MaterialProperty`. I have explained in here, https://github.com/idaholab/moose/pull/11358/commits/98f480d33d1471574e86b3f7f73da3d6c69575bf#diff-41e62cb55940db973a6410f7f52c1180R23

Reply:
It does not have to! It should be computed from the scalar variable and the node coordinates of the displacement variables. We cannot have constant monomial displacements!